data/campaigns/Heir_To_The_Throne/units/Human_Konrad_L1_Unarmed.cfg

```diff
@@ -0,0 +1,21 @@
+#textdomain wesnoth-h2tt
+
+[unit_type]
+    id=Unarmed_Fighter
+    name= _ "Unarmed Fighter"
+    race=human
+    num_traits=0
+    image="units/konrad/konrad-fighter-unarmed.png"
+    hitpoints=35
+    movement_type=smallfoot
+    movement=5
+    experience=50
+    level=1
+    alignment=lawful
+    advances_to=Captain
+    usage=mixed fighter
+    die_sound={SOUND_LIST:HUMAN_DIE}
+    hide_help=yes
+    cost=18
+    description= _ "Young and brash, Fighters attack with a sword, and are vulnerable to more seasoned enemies. However, they have the potential to become great warriors one day."
```

Comment:
I understand this is a copy of the normal unit description, but is there any connection to their unarmed status?